pkg/flowkit/state_test.go

```diff
@@ -137,6 +137,16 @@ func generateComplexProject() State {
 				HashAlgo:   crypto.SHA3_256,
 				PrivateKey: keys()[2],
 			},
+		}, {
+			Name:    "emulator-account-2",
+			Address: flow.HexToAddress("2c1162386b0a245f"),
+			Key: config.AccountKey{
+				Type:       config.KeyTypeHex,
+				Index:      0,
+				SigAlgo:    crypto.ECDSA_P256,
+				HashAlgo:   crypto.SHA3_256,
+				PrivateKey: keys()[1],
+			},
```

Comment:
Implemented additional account with same address, instead of creating another generatecomplexproject function